### ISSUE TITLE
Option -p should print sidecar containers resource usage

### ISSUE THREAD

**Issue by YvesEarnix** [robscott/kube-capacity#157]
Kubernetes v1.29 have introduce sidecar containers

https://kubernetes.io/docs/concepts/workloads/pods/sidecar-containers/

Sidecar containers are used as exemple by [istio](https://istio.io/latest/blog/2023/native-sidecars/) 

`kube-capacity -p ` should print sidecar containers resource usage

Note:
sidecar containers are visible with `kubectl top pods   --containers=true`

**Comment by ahmetb** [comment]
I also agree that this project should take `initContainers` as well as sidecar containers into account. There's non trivial calculation that goes into calculating what stage the container in, how much does that initContainer –or containers[]+sidecars[] request.
